cmd/health/main.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"flag"
+	"os"
+
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	"github.com/Kuadrant/multicluster-gateway-controller/pkg/apis/v1alpha1"
+	"github.com/Kuadrant/multicluster-gateway-controller/pkg/controllers/dnshealthcheckprobe"
+	"github.com/Kuadrant/multicluster-gateway-controller/pkg/health"
+	//+kubebuilder:scaffold:imports
+)
+
+var (
+	setupLog = ctrl.Log.WithName("setup")
+)
+
+func init() {
+	utilruntime.Must(clientgoscheme.AddToScheme(clientgoscheme.Scheme))
+
+	utilruntime.Must(v1alpha1.AddToScheme(clientgoscheme.Scheme))
+	//+kubebuilder:scaffold:scheme
+}
+
+func main() {
+	var metricsAddr string
+	var probeAddr string
+	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
+	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
```

Comment:
This will conflict with the defaults for the controller. Depends what the intention of this is whether that's an issue or not.

Reply:
I think we'll remove this file for now as we are running the probe controller alongide the MGC